icaruscode/TPC/Utilities/DetectorPropertiesICARUSClockOffsetMC.h

```diff
@@ -0,0 +1,294 @@
+////////////////////////////////////////////////////////////////////////
+// \file DetectorPropertiesICARUSClockOffsetMC.h
+//
+// \brief Goes with DetectorPropertiesICARUSClockOffsetMC.cxx ()
+//
+// \author brebel@fnal.gov
+//
+// Separation of service from Detector info class:
+// jpaley@fnal.gov
+//
+// Update for ICARUS by Bruce Howard and Jaesung Kim
+////////////////////////////////////////////////////////////////////////
+#ifndef DETINFO_DETECTORPROPERTIESICARUS_H
+#define DETINFO_DETECTORPROPERTIESICARUS_H
```

Comment:
Please use a full specification:
```suggestion
#ifndef ICARUSCODE_TPC_UTILITIES_DETECTORPROPERTIESICARUSCLOCKOFFSETMC_H
#define ICARUSCODE_TPC_UTILITIES_DETECTORPROPERTIESICARUSCLOCKOFFSETMC_H
```